axum/src/json.rs

```diff
@@ -196,6 +221,39 @@ where
     }
 }
 
+pub struct JsonSerializationError(serde_json::Error);
```

Comment:
I guess we only warn for missing docs on CI. Should probably be an error.

Reply:
Oh, right. This type also needs docs 😅